modules/preview/preview.php

```diff
@@ -169,3 +162,5 @@ public function get_link_expiration_options() {
 		]);
 	}
 }
+
+Preview::init();
```

Comment:
Like REST API endpoints we use already, the new modules `init()` themselves. Load order shouldn't be a huge problem for a couple of reasons:

- `vip-workflow.php` file determines which order files load. If we need to include a base class or something before other files, we can do it there.
- Otherwise, the `init()` function serves as an entry point to set filters on the WordPress lifecycle. These can use priorities, dependencies, and other WordPress concepts to manage load order. It's worth noting that `init()` is now called before any WordPress filters (as part of initial plugin loading), so we can now set a `'init'` action hook or any other hook at any priority without worrying about [the lower priority trick](https://github.com/Automattic/vip-workflow-plugin/pull/23#discussion_r1732150604) we used before.

Overall, this system has some nice properties. Each module is independent, and in charge of setting and removing their own hooks. Class intercommunication happens explicitly and functionally by calling static methods. There's no magic, and module logic is self-contained.